### ISSUE TITLE
about the label

### ISSUE THREAD

**Issue by xstju** [HongguLiu/MesoNet.Pytorch#3]
I want to train a model using the FaceForensics++ dataset , but I don't know how to set the label real/fake when training. Please help me.

**Comment by HongguLiu** [comment]
You can refer to the pytorch documents.[https://pytorch.org/docs/stable/torchvision/datasets.html#imagefolder](url)
